.gitignore

```diff
@@ -16,3 +16,4 @@ test/tmp
 test/version_tmp
 tmp
 lita_config.rb
+.idea
```

Comment:
I think this should go into your global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore